### PR TITLE
add missing step for file upload

### PR DESCRIPTION
**Pull request by ghost**
The readme.md was missing directions to upload an empty file with the name "terraform.tfstate". It is also helpful for readers to know that the SysLog server is optional and wouldn't break the deployment.

### CODE REVIEW

File: ci/pcfaws_terraform_pipeline.yml [hunk 1]
```diff
@@ -47,7 +47,7 @@ resources:
   type: git
   source:
     branch: master
-    uri: https://github.com/c0-ops/aws-concourse.git
+    uri: https://github.com/nsagoo-pivotal/aws-concourse.git
```
Comment (akumria) [inline ci/pcfaws_terraform_pipeline.yml:50]: was the change to your personal repo intended?

File: ci/pcfaws_terraform_pipeline.yml [hunk 2]
```diff
@@ -84,7 +84,7 @@ resources:
   source:
     api_token: {{PIVNET_TOKEN}}
     product_slug: elastic-runtime
-    product_version: 1\.9\..*
+    product_version: 1\.8\..*
```
Comment (akumria) [inline ci/pcfaws_terraform_pipeline.yml:87]: How come?

**Review by akumria — COMMENTED** [review]
Just wondering about the switch to your local repo.

**Comment by krishicks** [comment]
This should be fixed in 6dc6719b2ecdda32af0c0e95ce87b40cdc8b3e4d.

**Pull request closed without merging**